src/test/java/com/trainigcenter/springtask/service/impl/ActorServiceImplTest.java

```diff
@@ -0,0 +1,134 @@
+package com.trainigcenter.springtask.service.impl;
+
+import com.trainigcenter.springtask.domain.Actor;
+import com.trainigcenter.springtask.domain.Genre;
+import com.trainigcenter.springtask.domain.Movie;
+import com.trainigcenter.springtask.persistence.ActorRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ActorServiceImplTest {
+
+    @Mock
+    private ActorRepository actorRepository;
+
+    @InjectMocks
+    private ActorServiceImpl actorService;
+
+    private Actor robertDowney;
+    private Actor chrisEvans;
+    private Actor chrisHemsworth;
+    private List<Actor> actors;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        robertDowney = new Actor();
+        robertDowney.setId(1);
+        robertDowney.setName("Robert Downey Jr.");
+        robertDowney.setBirthYear(1965);
+
+        chrisEvans = new Actor();
+        chrisEvans.setId(2);
+        chrisEvans.setName("Chris Evans");
+        chrisEvans.setBirthYear(1981);
+
+        chrisHemsworth = new Actor();
+        chrisHemsworth.setId(3);
+        chrisHemsworth.setName("Chris Hemsworth");
+        chrisHemsworth.setBirthYear(1983);
+
+        actors = List.of(robertDowney, chrisEvans, chrisHemsworth);
+    }
+
+    @Test
+    public void getAllTest() {
+        when(actorRepository.findAll()).thenReturn(actors);
+        List<Actor> genresActual = actorService.getAll();
+
+        assertEquals(genresActual, actors);
+    }
+
+    @Test
+    public void getByIdTest() {
+        when(actorRepository.findById(robertDowney.getId())).thenReturn(Optional.of(robertDowney));
+        Actor actor = actorService.getById(robertDowney.getId()).get();
+
+        assertNotNull(actor);
```

Comment:
By this assertion you check only that service returns something that non-null. You don't actually test it's behavior and data it returns. It is valid all the tests you wrote and all of the tests should revised. Materials by this link can give you more ideas about what is unit testing: https://habr.com/ru/post/169381/